app/app.go

```diff
@@ -1034,6 +1047,12 @@ func (app *StrideApp) RegisterTendermintService(clientCtx client.Context) {
 	tmservice.RegisterTendermintService(clientCtx, app.BaseApp.GRPCQueryRouter(), app.interfaceRegistry, app.Query)
 }
 
+// RegisterNodeService registers the node gRPC Query service.
+func (app *StrideApp) RegisterNodeService(clientCtx client.Context) {
+	// TODO: implement!
```

Comment:
what's this?

Reply:
Let's ask Notional

Reply:
it only for implement the servertypes.Application interface method

Reply:
do we need it?

Reply:
Yes we need it, this is implementation func of sdk `Application` interface

Reply:
do we actually have to implement it?

Reply:
no, just need that as empty func would work as well

Reply:
done